### PR TITLE
Refactors and rebalances door charges

### PR DESCRIPTION
**Pull request by Cyberboss**
Closes #27614

:cl:
balance: Door charge cost reduced to 1 TC from 2
balance: Door charges no longer ignite victims
balance: Door charge damage changed from 40 brute + heavy explosion to 40 brute 20 burn
balance: Door charges changed from 16 seconds of paralysis to 16 seconds of stun
fix: Door charges now affect non-humans
tweak: Airlock maintenance panels can no longer be destroyed
/:cl:

More of a meme tool now, but still better than instant K.O.

### CODE REVIEW

**Comment by ChangelingRain** [comment]
this is still not an improvement because it's not a fun object, it's an object you stick on a door and it just randomly fucks people, that is the opposite of fun

**Comment by Armhulen** [comment]
I think this is good, you can use it to ambush people and set up traps.

File: code/game/machinery/doors/airlock.dm [hunk 1]
```diff
@@ -201,9 +200,7 @@
 /obj/machinery/door/airlock/Destroy()
 	qdel(wires)
 	wires = null
```
Comment (ChangelingRain) [inline code/game/machinery/doors/airlock.dm]: get ye QDEL_NULL()

Comment (Cyberboss) [inline code/game/machinery/doors/airlock.dm]: >prs should be atomic

Comment (Cyberboss) [inline code/game/machinery/doors/airlock.dm]: #27668

**Comment by kevinz000** [comment]
👍 fuck old door charges being a one hit kill.

File: code/game/machinery/doors/airlock.dm [hunk 2]
```diff
@@ -201,9 +200,7 @@
...
-	if(charge)
-		qdel(charge)
-		charge = null
+	qdel(charge)
 	if(electronics)
 		qdel(electronics)
```
Comment (ChangelingRain) [inline code/game/machinery/doors/airlock.dm]: QDEL_NULL

File: code/game/machinery/doors/airlock.dm [hunk 3]
```diff
@@ -201,9 +200,7 @@
 /obj/machinery/door/airlock/Destroy()
 	qdel(wires)
 	wires = null
-	if(charge)
-		qdel(charge)
-		charge = null
+	qdel(charge)
```
Comment (ChangelingRain) [inline code/game/machinery/doors/airlock.dm]: QDEL_NULL

Comment (Cyberboss) [inline code/game/machinery/doors/airlock.dm]: `charge` nulls itself in it's destroy

File: code/game/machinery/doors/airlock.dm (hunk 2 above)
Comment (Cyberboss) [inline code/game/machinery/doors/airlock.dm]: #27678

**Pull request closed without merging**